### PR TITLE
chore: kramdown plugin extension supporting code-excerpts

### PR DESCRIPTION
**Pull request by chalin**
Staged at: https://angulardart-org-dev.firebaseapp.com/playground

Contributes to
- #637 - in helping us transition from Jade to markdown
- #374 - [ng] fix makeExample stylePatterns

Among other variants, it supports markdown like:

    <?code-excerpt "quickstart/lib/app_component.dart" title?>
    ```
      import 'package:angular2/angular2.dart';

      [!@Component(
        selector: 'my-app',
        template: '<h1>Hello {{name}}</h1>')!]
      class AppComponent {
        var [!name!] = 'Angular';
      }
    ```
And renders it like this:
> screen shot 2017-05-19 at 4 54 07 pm

### CODE REVIEW

File: src/playground/jade-test.jade [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (kwalrath) [inline src/playground/jade-test.jade]: This file doesn't seem to work... but I think you know that.

File: src/playground/index.md [hunk 2]
```diff
@@ -1,27 +1,83 @@
 ---
-layout: default
-title: "CE Playground"
-short-title: "CE Playground"
-description: "A playground to test web components."
+layout: angular
+title: "Writer's Playground"
```
Comment (kwalrath) [inline src/playground/index.md]: I'd say **Writers'**, unless you don't want me to play there. :)

File: src/playground/jade-test.jade (hunk 1 above)
Comment (chalin) [inline src/playground/jade-test.jade]: Right, that's the next thing to tackle.

File: src/playground/index.md (hunk 2 above)
Comment (chalin) [inline src/playground/index.md]: lol ... of course you can! :)

**Pull request merged**